README.md

```diff
@@ -91,6 +91,56 @@ sudo apt-get install protobuf-compiler libprotoc-dev
 pip install onnx
 ```
 
+### Windows
+When building on Windows it is highly recommended that you also build protobuf locally as a static library. The version distributed with conda-forge is a DLL and this is a conflict as ONNX expects it to be a static lib.
+
+#### Instructions to build protobuf and ONNX on windows
+Step 1 : Build protobuf locally
+```
+git clone https://github.com/protocolbuffers/protobuf.git
+git checkout 3.9.x
+cd cmake
+cmake -G "Visual Studio 15 2017 Win64" -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_BUILD_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=<protobuf_install_dir>
+msbuild protobuf.sln /m /p:Configuration=Release
+msbuild INSTALL.vcxproj /p:Configuration=Release
+```
+
+Step 2: Build ONNX
+```
+# Get ONNX
+git clone https://github.com/onnx/onnx.git
+cd onnx
+git submodule update --init --recursive
+
+# Set environment variables to find protobuf
+set PATH=<protobuf_install_dir>\bin;%PATH%
+
```

Comment:
I would suggest adding this path to the user environment variables so it's more permanent. Otherwise it's easily forgotten and the next time they try to rebuild they can get errors and not remember that a custom protobuf directory needed to be added to the path.